jme3-core/src/test/java/com/jme3/math/QuatMultTest.java

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright (c) 2009-2015 jMonkeyEngine
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are
+ * met:
+ *
+ * * Redistributions of source code must retain the above copyright
+ *   notice, this list of conditions and the following disclaimer.
+ *
+ * * Redistributions in binary form must reproduce the above copyright
+ *   notice, this list of conditions and the following disclaimer in the
+ *   documentation and/or other materials provided with the distribution.
+ *
+ * * Neither the name of 'jMonkeyEngine' nor the names of its contributors
+ *   may be used to endorse or promote products derived from this software
+ *   without specific prior written permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED
+ * TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR
+ * PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR
+ * CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+ * EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+ * PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
+ * PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF
+ * LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
+ * NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+ * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+package com.jme3.math;
+
+import java.util.Random;
+import org.junit.*;
+import static org.junit.Assert.*;
+import static com.jme3.math.FastMath.*;
+
+public class QuatMultTest {
+    Random rand;
+    @Before
+    public void setUp() {
+        rand = new Random(0);
+    }
+    
+    @After
+    public void tearDown() {
+    }
+   
+    @Test
+    public void test() {
+        Quaternion quat = new Quaternion();
+        Vector3f vect = new Vector3f();
+        Vector3f store60 = new Vector3f(); //stores result of the current implementation
+        Vector3f store15 = new Vector3f(); //stores result of the proposed change
+
+//        final int reps = 100000;
+        final int reps = 10;
+         
+        for(int i = 0; i < reps; i++) {
+            randQuat(quat);
+            
+            randVect(vect);
+            
+            quat.mult(vect, store60);
+            mult15V2(quat, vect, store15);
+    
+            //eps stands for the epsillon in the approximateEquals method
+            //set your desired eps
+            
+            //final float eps = 0.00001f; 
+            //final float eps = 0.001f; 
+            final float eps = 0.08f;
```

Comment:
Is it intended to use such a high epsilon value? Is it failing on the default value in `FastMath.approximateEquals`?